Gemfile

```
@@ -4,15 +4,9 @@ source 'https://rubygems.org'
 
 ruby RUBY_VERSION
 
-# Force gem rails to 5.2.2.1 to fix some vulnerabilities
-# on actionview and railties
-# It can be removed when new stable version will be released or
-# when Decidim force the rails version
-gem 'rails', '5.2.2.1'
-
 gemspec
 
-gem 'decidim', "~> 0.16.0"
+gem 'decidim', ">= 0.17.0"
```

To maximize compatibility the change should be 
```suggestion
gem 'decidim', ">= 0.16.0"
```